src/nise/examples/nise_basic_example.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+import argparse
+import nise
+import sys
+
+
+def example(virtual_device_name, connection):
+    print('Opening session to', virtual_device_name)
+    with nise.Session(virtual_device_name=virtual_device_name) as session:
+        print('Connecting', connection)
+        session.connect(connection)
+        print(connection, ' is now connected.')
```

Comment:
I'd remove all prints but this "result" one.

Reply:
fixed